src/github.com/matrix-org/dendrite/clientapi/routing/memberships.go

```diff
@@ -30,6 +30,10 @@ type response struct {
 	Chunk []gomatrixserverlib.ClientEvent `json:"chunk"`
 }
 
+type joinedResponse struct {
+	JoinedRooms []string `json:"joined_rooms,flow"`
```

Comment:
What's the flow tag mean?

Reply:
It's something used by yaml, not json. Will remove.